File: 1-Iniciante/python/1064_positivos-e-media.py
numeros = []
ns, positivos = 0, 0
for x in range(6):
    numeros.append(float(input()))
for numero in numeros:
    if numero > 0: 
        ns += 1
        positivos += numero
print(f'{ns} valores positivos\n{(positivos/ns):.1f}')

File: 1-Iniciante/c/1064_positivos-e-media.c
#include <stdio.h>

int main () {
    float y, positivos = 0;
    int ns = 0;
    for (int x = 0; x < 6; x++) {
        scanf("%f", &y);
        if (y > 0) {
            ns++;
            positivos =+ y;
        }
    }
    printf("%d valores positivos\n%.1f\n", ns, positivos/ns);
    return 0;
}